api/server/controllers/auth.controller.js

```diff
@@ -107,60 +65,60 @@ const resetPasswordController = async (req, res) => {
   }
 };
 
-const refreshController = async (req, res, next) => {
-  const { signedCookies = {} } = req;
-  const { refreshToken } = signedCookies;
-  //TODO
-  // if (refreshToken) {
-  //   try {
-  //     const payload = jwt.verify(refreshToken, process.env.REFRESH_TOKEN_SECRET);
-  //     const userId = payload._id;
-  //     User.findOne({ _id: userId }).then(
-  //       (user) => {
-  //         if (user) {
-  //           // Find the refresh token against the user record in database
-  //           const tokenIndex = user.refreshToken.findIndex(item => item.refreshToken === refreshToken);
+// Currently not in use
+const refreshController = async () => {}
```

Comment:
Note that refresh token support is on the task list but there are so many other to-do's that I wouldn't have been able to get to it for a while, so if you want to take that on, it would be incredibly awesome. The challenge with implementing refresh tokens is mostly on the frontend for handling token exchange IMO, but it would be great to get the access token AWAY from being set as a cookie.